fix: fix the starter kit issue caused by knn

zach-shu opened the pull request:
When sending a search query to Elasticsearch, `query` and `knn` parameters cannot be used at the same time unless it is a hybrid search. This PR removes the `knn` parameter assignment in the actions JSON to make the starter kit work out of the box. 

Slack thread: https://ibm-analytics.slack.com/archives/C1U97HQ3T/p1711996051841799?thread_ts=1711755097.522389&cid=C1U97HQ3T

rfazeli reviewed (commented):
Thanks, Zach! This looks good to me. Is this addressing all the issues on the thread or just the first one?

zach-shu commented:
> Thanks, Zach! This looks good to me. Is this addressing all the issues on the thread or just the first one?

Thanks, Reza, for your review. This is to address the first one. The first one is more critical IMO.

sne3091 reviewed (approved):
Looks good! Thanks Zach
